webui/react/src/routes/utils.ts

```diff
@@ -1,23 +1,33 @@
 import { pathToRegexp } from 'path-to-regexp';
 import { MouseEvent, MouseEventHandler } from 'react';
 
+import { globalStorage } from 'globalStorage';
 import history from 'routes/history';
 import { Command, CommandType } from 'types';
 import { clone } from 'utils/data';
 
 import routes from './routes';
+import { RouteConfig } from './types';
 
-export const serverAddress = (avoidDevProxy = false, path = ''): string => {
-  if (!!path && !isAbsolutePath(path)) return path;
+// serverAddress returns determined cluster (master) address.
+export const serverAddress = (path = ''): string => {
```

Comment:
So nice to not to have to worry about passing in `avoidDevProxy`!

Reply:
very nice refactor in `routes/utils.ts`!

Reply:
thanks